frog/imports/ui/ErrorPage/index.stories.js

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import ErrorPage from '.';
+
+storiesOf('Error Page', module).add('Graph', () => (
+  <PageNotFound title="404 Graph" message="This graph does not exist" />
```

Comment:
`PageNotFound` is not defined. I think you meant to use the component `ErrorPage`

Reply:
Ah yes! My bad.